composer.json

```diff
@@ -10,36 +10,36 @@
         }
     ],
     "require": {
-        "php": "^8.0 || ^8.1",
+        "php": "^8.1 || ^8.2",
```

Comment:
It would be better to use `~8.1 || ~8.2` because having `^8.1` also includes `8.2` and `8.3` etc. but we can't know for sure if 8.3 would break anything.